src/renderer/services/deposit/utils.ts

```diff
@@ -52,7 +55,7 @@ export function generateDeposit(signingKey: KeyPair, withdrawalPubKey: BLSPubKey
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     const amount: BN = new BN(
         // remove decimal zeroes
-        utils.formatUnits(utils.parseEther(depositAmount), "gwei").split(".")[0]
+        utils.formatUnits(utils.parseEther(depositAmount.toString()), "gwei").split(".")[0]
```

Comment:
gwei?

Reply:
yes, deposit contract and eth2 handles everything in gweis